package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-saga-test-plan",
-  "version": "4.0.0-beta.2",
+  "version": "4.0.0-beta.3",
```

Comment:
Did `4.0.0-beta.2` get pushed to npm? As I don't see it.

Reply:
Nope. So we should stay on beta.2?